build.gradle

```diff
@@ -90,22 +84,39 @@ dependencies {
     implementation "org.grails:grails-plugin-url-mappings"
     implementation "org.grails:grails-plugin-interceptors"
     implementation "org.grails.plugins:cache"
+    implementation "org.grails.plugins:cache-ehcache:3.0.0"
     implementation "org.grails.plugins:async"
     implementation "org.grails.plugins:scaffolding"
     implementation "org.grails.plugins:events"
-    implementation "org.grails.plugins:hibernate5"
-    implementation "org.hibernate:hibernate-core:5.4.18.Final"
+    implementation "org.grails.plugins:hibernate5:${gormVersion}"
+    implementation "org.hibernate:hibernate-core:5.6.15.Final"
+    implementation "org.hibernate:hibernate-ehcache:5.4.9.Final"
+    implementation "org.grails.plugins:scaffolding"
     implementation "org.grails.plugins:gsp"
+    implementation "org.codehaus.groovy:groovy-all:your-groovy-version"
+    implementation "io.micronaut:micronaut-core:3.10.2"
+    implementation "io.micronaut:micronaut-inject:3.10.2"
     compileOnly "io.micronaut:micronaut-inject-groovy"
     console "org.grails:grails-console"
     implementation "org.grails:grails-plugin-filters:3.0.12"
     implementation "org.grails:grails-datastore-rest-client:6.1.12.RELEASE"
     profile "org.grails.profiles:web"
-    runtimeOnly "org.glassfish.web:el-impl:2.1.2-b03"
+    runtimeOnly "org.glassfish.web:el-impl:2.2.1-b05"
     runtimeOnly "com.h2database:h2"
     runtimeOnly "org.apache.tomcat:tomcat-jdbc"
+    runtimeOnly "javax.xml.bind:jaxb-api:2.3.1"
```

Comment:
Dependency defined elsewhere, e.g. grails-plugin-rest. Is there a reason to add it?